internal/databases/postgres/backup_list_handler_test.go

```diff
@@ -1,13 +0,0 @@
-package postgres_test
-
-import (
-	"testing"
-
-	"github.com/wal-g/wal-g/internal/databases/postgres"
-	"github.com/wal-g/wal-g/testtools"
-)
-
-func TestBackupListFlagsFindsBackups(t *testing.T) {
```

Comment:
This is not a joke, this test is actually no longer needed because the default internal.DefaultHandleBackupList now accept the flags.